scripts/run-tests.js

```
@@ -0,0 +1,131 @@
+    success = runTest(
+      'Models (Mock)', 
+      'npx mocha "test-models/**/*.test.js"', 
+      { ...testEnv, USE_MOCK_RESPONSE: 'true' }
+    );
+    break;
+    
+  case 'models':
+    console.log(`\n${colors.bright}${colors.yellow}⚠️  Warning: This will hit the real API${colors.reset}\n`);
+    const proceed = process.env.CI === 'true' || process.env.FORCE_MODELS === 'true';
+    
+    if (proceed) {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)